docs/style_guide/README.md

```diff
@@ -0,0 +1,78 @@
+# Ballerina Style Guide
+
+Formatting is one of the most argued topic when it comes to a programming language.
+To avoid these arguments we decide to come up with a opinionated guide for Ballerina source code formatting. 
+Users can follow their own formatting but we recommend to use this formatting guide as all the IDE tools 
+will support only this format. Also following this guide will form a standard style across all developers.
+
+## Formatting Conventions
+
+### Indentation and line length
+* Use four spaces, not tabs
+* No indentation top level (definitions) 
```

Comment:
Shall we change this to, **_No indentation for top level definitions_** and give a couple of example definitions such as function and service

Reply:
Will do the change..